### PR TITLE
added hero layout

### PR DESCRIPTION
**Pull request by igorivnaolga**

### CODE REVIEW

File: src/css/hero.css [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+  gap: 40px;
+  justify-content: flex-end;
+  margin: 0 auto;
+
+  background-repeat: no-repeat;
+  background-position: center;
+  border-radius: 16px;
+  background:
+    linear-gradient(357deg, rgba(0, 0, 0, 0.4) 39.23%, rgba(0, 0, 0, 0) 66.28%),
+    url(<path-to-image>),
```
Comment (asaulyak) [inline src/css/hero.css:17]: specify path the image
add image to the `img` folder
add images for different resolutions and dpr

File: src/css/hero.css [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+    lightgray -566px -29.886px / 332.537% 104.761% no-repeat;
+  background-size: cover;
```
Comment (asaulyak) [inline src/css/hero.css:19]: background styles are overwritten

File: src/css/hero.css [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+}
+.container {
+  position: relative;
```
Comment (asaulyak) [inline src/css/hero.css:22]: remove

File: src/css/hero.css [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+  color: var(--white);
+  font-size: 38px;
+  font-weight: 700;
+  line-height: 1.05;
+}
+.hero-header-english {
+  color: var(--soft-green);
+}
+.hero-button {
+  max-width: 295px;
```
Comment (asaulyak) [inline src/css/hero.css:36]: use global `button` styles

File: src/css/hero.css [hunk 5]
```diff
@@ -0,0 +1,110 @@
...
+  font-weight: 700;
+  line-height: 1.71;
+
+  color: var(--white);
+  background-color: var(--green);
+
+  transition: background-color var(--animation-fast);
+}
+.hero-button:hover {
+  background-color: #38ae5f;
```
Comment (asaulyak) [inline src/css/hero.css:53]: use variables

File: src/css/hero.css [hunk 6]
```diff
@@ -0,0 +1,186 @@
...
+  margin-left: auto;
+  padding: 0;
+
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
+  width: 38px;
+  height: 38px;
+  border-radius: 50px;
```
Comment (asaulyak) [inline src/css/hero.css:122]: 50%

File: src/css/hero.css [hunk 7]
```diff
@@ -0,0 +1,186 @@
...
+}
+.scroll-to-bottom-icon:hover {
+  fill: var(--white);
+}
+@media screen and (min-width: 768px) {
+  .hero {
+    padding-bottom: 100px;
+  }
+  .hero .hero-container {
+    max-width: 704px;
```
Comment (asaulyak) [inline src/css/hero.css:143]: min-width is taken from the global container style

File: src/css/hero.css [hunk 8]
```diff
@@ -0,0 +1,186 @@
...
+  color: var(--white);
+  font-size: 38px;
+  font-weight: 700;
+  line-height: 1.05;
+}
+.hero-header-english {
+  color: var(--soft-green);
+}
+
+.hero-buttons-wrapper {
```
Comment (asaulyak) [inline src/css/hero.css:93]: `.hero-buttons-wrapper` is not needed here
scroll-to-bottom button should be absolute positioned with `bottom` and `right` values

File: src/css/hero.css [hunk 9]
```diff
@@ -0,0 +1,185 @@
...
+    background-image: linear-gradient(
+        357deg,
+        rgba(0, 0, 0, 0.4) 39.23%,
+        rgba(0, 0, 0, 0) 66.28%
+      ),
+      url('../img/hero/bg-desktop@2x.jpg');
+  }
+}
+.hero-container {
+  padding: 20px;
```
Comment (asaulyak) [inline src/css/hero.css:78]: combine paddings in one line

File: src/css/hero.css [hunk 10]
```diff
@@ -0,0 +1,185 @@
...
+}
+@media screen and (min-width: 768px) {
+  .hero {
+    max-width: 768px;
+    padding-bottom: 100px;
+    min-height: 857px;
+    padding: 32px;
+  }
+  .hero-container {
+    padding: 0;
```
Comment (asaulyak) [inline src/css/hero.css:151]: combine paddings in one line

File: src/partials/hero.html [hunk 11]
```diff
@@ -1 +1,19 @@
-<section>hero</section>
+<section class="hero section">
+  <div class="container">
+    <div class="hero-container">
```
Comment (asaulyak) [inline src/partials/hero.html:3]: add `hero-container` class to the parent div
like this
`<div class="container hero-container">`

File: src/css/hero.css (hunk 5 above)
Comment (igorivnaolga) [reply src/css/hero.css:103]: I cant find this color in variables. In main styles also used this color without var

Comment (asaulyak) [reply src/css/hero.css:106]: add color if it's missing

File: src/css/main.css [hunk 12]
```diff
@@ -146,7 +146,11 @@
 }
 
 .button:hover {
+<<<<<<< HEAD
```
Comment (asaulyak) [inline src/css/main.css:149]: remove this line

File: src/css/main.css [hunk 13]
```diff
@@ -146,7 +146,11 @@
...
+  background-color: var(--pressed-green);
+=======
```
Comment (asaulyak) [inline src/css/main.css:151]: remove this line

File: src/css/main.css [hunk 14]
```diff
@@ -146,7 +146,11 @@
...
   background-color: var(--gray);
+>>>>>>> 471daa3168117c941178f9db2fd194d0269d59c1
```
Comment (asaulyak) [inline src/css/main.css:153]: remove this line

File: src/css/variables.css [hunk 15]
```diff
@@ -2,7 +2,11 @@
   --black: #121417;
   --soft-black: rgba(18, 20, 23, 0.5);
   --green: #4dc274;
+<<<<<<< HEAD
```
Comment (asaulyak) [inline src/css/variables.css:5]: remove this line

File: src/css/variables.css [hunk 16]
```diff
@@ -2,7 +2,11 @@
...
+  --pressed-green: #38ae5f;
+=======
```
Comment (asaulyak) [inline src/css/variables.css:7]: remove this line

File: src/css/variables.css [hunk 17]
```diff
@@ -2,7 +2,11 @@
...
   --ultra-soft-green: rgba(77, 194, 116, 0.1);
+>>>>>>> 471daa3168117c941178f9db2fd194d0269d59c1
```
Comment (asaulyak) [inline src/css/variables.css:9]: remove this line

**Review by asaulyak — APPROVED** [review]

**Pull request merged**